css/index.css

```diff
@@ -1 +1,413 @@
\ No newline at end of file
+/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+html{
+    font-size: 62.5%;
+}
+
+p{
+    line-height: 1.2em;
+    font-size: 1.6rem;
+
+}
+ul{
+	margin-top: 20px;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+    font-size: 1.6rem;
+}
+
+.container {
+  max-width: 880px;
+  width: 100%;
+  margin: 0 auto;
+}
+.middle-img{
+    width: 100%;
+    margin: 0 auto;
+}
+
+/* header */
+header {
+	display: flex;
+	justify-content: space-between;
+}
+
+header nav {
+
+  width: 600px;
+  margin-top: 35px;
+	display: flex;
+	justify-content: space-between;
+}
+
+header nav a {
+  font-size: 1.4rem;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+
+  margin-top: 25px;
+}
+
+/* cta */
+.cta{
+    display: flex;
+    justify-content: flex-end;
+
+}
+.cta h1{
+    font-size: 7rem;
+}
+
+.cta .cta-text {
+  vertical-align: top;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+
+  margin-top: 69px;
+}
+
+main {
+  border-top: 1px solid black;
+  margin: 50px 10px 0 10px;
+  padding-top: 0;
+}
+
+main .top-content {
+  padding: 0 ;
+    display: flex;
```

Comment:
You have some spacing and indentation inconsistencies; remember to try out [prettier](https://prettier.io/) as a remedy!